File: Day 030/Homework/homework7.py
def numbers():
    nums = []
    
    for i in range(10):
        while True:
            try:
                num = float(input("Enter any number: "))
                nums.append(num)
                break
            except ValueError:
                print("Enter valid number")
    
    for num in nums:
        square = num ** 2
        print(num, "is in square of", square)

numbers()

File: Day 084/Homework/homework7.js
// 7. მომხმარებელს შემოატანინეთ რიცხვი. და შეამოწმეთ იყოფა თუ არა 3-ზე ან 2-ზე მაინც

let num = Number(prompt("Enter any number: "))

console.log(num % 3 == 0 || num % 2 == 0)